app/backend/src/couchers/servicers/communities.py

```diff
@@ -99,9 +96,10 @@ def ListCommunities(self, request, context):
             next_node_id = int(request.page_token) if request.page_token else 0
```

Comment:
The pagination needs to be based on the name as well: this will still sort within each page only